tests/nix_tests/test_no_abci_resp.py

```diff
@@ -16,14 +16,41 @@ def evmos(tmp_path_factory):
     )
 
 
-def test_gas_eth_tx(evmos):
+@pytest.fixture(scope="module")
+def custom_evmos_rocksdb(tmp_path_factory):
+    path = tmp_path_factory.mktemp("no-abci-resp-rocksdb")
+    yield from setup_custom_evmos(
+        path,
+        26810,
+        memiavl_config(path, "discard-abci-resp"),
+        post_init=create_snapshots_dir,
+        chain_binary="evmosd-rocksdb",
+    )
+
+
+@pytest.fixture(scope="module", params=["evmos", "evmos-rocksdb"])
+def evmos_cluster(request, custom_evmos, custom_evmos_rocksdb):
+    """
+    run on evmos and
+    evmos built with rocksdb (memIAVL + versionDB)
+    """
+    provider = request.param
+    if provider == "evmos":
+        yield custom_evmos
+    elif provider == "evmos-rocksdb":
+        yield custom_evmos_rocksdb
+    else:
+        raise NotImplementedError
```

Comment:
This function is repeated in multiple files. I see that this is because there is a different `custom_evmos_rocksdb` function and `custom_evmos` defined in the corresponding files, but maybe there is a way to have the `evmos_cluster` be defined session-wide and then pass the respective fixtures to it at the current scope where the test is executed?

Reply:
The problem of doing that is that you'll have the same chain running thruout all tests. This is not always desired. So if we need a fresh chain with a specific config, we need to define a scoped cluster for each test,